addons/website_payment/models/payment_provider.py

```diff
@@ -48,3 +48,15 @@ def copy(self, default=None):
         if self._context.get('stripe_connect_onboarding'):
             res.website_id = False
         return res
+
+    def _is_tokenization_required(self, **kwargs):
```

Comment:
This works, but rather than overriding `_is_tokenization_required` and returning `True` when it is *not* allowed, I think it would save future headaches to override `_compute_show_tokenize_input_mapping` instead, until the day we have a `_is_tokenization_allowed` method to override.